src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -197,8 +174,6 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
         display: 'flex',
       }),
 
-      ...itemSeparator({ props, variables: v, theme }),
```

Comment:
I am not sure that we want to remove separators in Teams theme menu

Reply:
I had the same doubt, but wasn't sure how can I show the separator. Will add example with some other menu type that does not have separator by default then... Thanks for the suggestion.

Reply:
Addressed the comment, added example for some other variants, take a look in the description of the PR.